Docker-compose/docker-compose.yml

```diff
@@ -110,8 +108,7 @@ services:
     ports:
       - "10000:10000"
     volumes:
-       - /usr/share/zoneinfo/Europe/Rome:/etc/localtime:ro
-       - ../docker-example:/satosa_proxy:rw
+       - ./satosa:/satosa_proxy:rw
```

Comment:
satosa -> project?

I agree that `docker-example` is too much related to the demo purposes, while satosa is too much generic and might confuse the software/application name with the local configuration, I suppose that `project` or any other similar alternative might better explain the purpose of that folder